index.js

```diff
@@ -12,13 +12,18 @@ module.exports =
         /* Decls are the properties inside */
         rule.walkDecls(decl => {
           let value = decl.value;
+          let important = decl.important;
 
           // Checking that declaration has `vh` and is not corrected.
           let isMatch = value.match(finderRegex) !== null;
           let isPreParsed = value.match(excludeRegex) === null;
           if (isMatch && isPreParsed) {
             let correctedViewport = value.replace(finderRegex, replaceBy);
 
+            if (important) {
+              correctedViewport += ' !important';
```

Comment:
Is there any other way to get this directive working? i.e. You used `decl.important` to get isImportant but then used string to specify directive.

Reply:
We can use the raw value instead

Reply:
How? can I get some link or code changes.

Reply:
We get this structure : 
```json 
{
  "raws": {
    "before": " ",
    "between": ": ",
    "important": " ! important"
  },
  "type": "decl",
  "source": {
    "start": {
      "line": 1,
      "column": 29
    },
    "input": {
      "css": "a{ height: 10vh !important; max-height: 20vh ! important; color: #FF10GG; }",
      "hasBOM": false,
      "id": "<input css 7>"
    },
    "end": {
      "line": 1,
      "column": 57
    }
  },
  "prop": "max-height",
  "important": true,
  "value": "20vh"
}

```

Reply:
Do you want an updated PR with the raw important property ?

Reply:
In fact, if the important directive is ` !important` (spaces are important here),  the raws.important property do not exists :
```json
{
  "raws": {
    "before": " ",
    "between": ": "
  },
  "type": "decl",
  "source": {
    "start": {
      "line": 1,
      "column": 4
    },
    "input": {
      "css": "a{ height: 10vh !important; max-height: 20vh ! important; color: #FF10GG; }",
      "hasBOM": false,
      "id": "<input css 7>"
    },
    "end": {
      "line": 1,
      "column": 27
    }
  },
  "prop": "height",
  "important": true,
  "value": "10vh"
}

```

Reply:
So we can use  `decl.raws.important || ' !important'`

Reply:
In fact, post css use this trick too: 
https://github.com/postcss/postcss/blob/21ce39c7aa19f1b0b237a5a180a59bf786111fdf/lib/stringifier.es6#L45

Reply:
Force pushed to use the same way than postcss